stylesheets/app.scss

```diff
@@ -201,10 +202,9 @@ button, .button {
     margin-left: 20px;
   }
   &.blue {
-    background:$med-blue;
-    &:hover {
-      background: lighten($med-blue,10%);
-    }
+    background:white;
```

Comment:
This is going to affect ALL buttons and links that look like buttons, is that the intended effect?